### DISCUSSION TITLE
Stack "Open in Terminal" with already open terminal

### DISCUSSION THREAD

**Discussion by rvnexe** [microsoft/terminal#15759] [Q&A] [answered]
Hello. When you click "Open in Terminal" in right click menu, The terminal opens that directory.
Then if you do that again there/somewhere else, A New Windows opens.
Shouldn't it make just a new tab in the previous terminal?

**Comment by zadjii-msft** [comment, marked as answer]
This'll obey your [`windowingBehavior`](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/startup#new-instance-behavior) setting. If you change that to "Attach to the most recently used window"
![image](https://github.com/microsoft/terminal/assets/18356694/960a5a7e-790a-49ce-adf6-85ab653333ad)

then the explorer context menu should open in the previous Terminal window.

**Reply by rvnexe** [reply]
Cheers!
